### ISSUE TITLE
Issue with building cross tool chain using Bootstrap toolchain from cross-compiled source method

### ISSUE THREAD

**Issue by sekharpedamallu** [golang-fips/go#77]
I m trying to build  FIPS patched go code using  **Bootstrap toolchain from cross-compiled source method** documented at 
 **(https://go.dev/doc/install/source#bootstrapFromCrosscompiledSource ).**

I'm seeing below problem:
# GOOS=linux GOARCH=amd64 ./bootstrap.bash
#### Building ../../go-linux-amd64-bootstrap
Building Go cmd/dist using /usr/local/go. (go1.19.4 linux/amd64)
Building Go toolchain1 using /usr/local/go.
Building Go bootstrap cmd/go (go_bootstrap) using Go toolchain1.
Building Go toolchain2 using go_bootstrap and Go toolchain1.
Building Go toolchain3 using go_bootstrap and Go toolchain2.
Building packages and commands for linux/amd64.
vendor/github.com/golang-fips/openssl-fips/openssl
In file included from /workdir/openssl-fips/go-linux-amd64-bootstrap/src/vendor/github.com/golang-fips/openssl-fips/openssl/aes.go:10:
.**/goopenssl.h:15:10: fatal error: openssl/ossl_typ.h: No such file or directory
   15 | #include <openssl/ossl_typ.h>
      |          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
go tool dist: FAILED: /workdir/openssl-fips/go-linux-amd64-bootstrap/pkg/tool/linux_amd64/go_bootstrap install std cmd: exit status 2**

Basically the issue looks to be with goopenssl.h not able to find the openssl in the standard header paths.
I m having openssl include's at /usr/local/ssl/include/openssl path, how can the compiler find the /usr/local/ssl/include/openssl header path's so that it can be compiled ?? Any help appreciated.

**Comment by sekharpedamallu** [comment]
Could figure out the issue for setting openssl libraries to be part of standard headers/libraries path.
